### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by agiledivider**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,17 +16,19 @@
...
                 }
             }
         })
     })
 
-    async function underline() {
-        let result = await miro.board.selection.get();
+    async function underline(widgets) { // accept widgets as parameter, work on te
+        //let result = await miro.board.selection.get();
+        let result = widgets;
```
Comment (agiledivider) [inline index.html:31]: Have not tried it though.

**Comment by Pheels** [comment]
Still the same issue I am afraid. I believe that getWidgetMenuItems is extended before the state of the widget is resolved, which might make this impossible.
Screenshot 2020-05-14 at 12 00 25

**Comment by agiledivider** [comment]
But i see your underline icon, just like expected, right?
And since you do not have any text, it cannot replace anything.

**Comment by Pheels** [comment]
The issue on the question was that I wanted the button to display in the case that the widget was not locked. Apologies if that wasn't clear.

**Comment by agiledivider** [comment]
> The issue on the question was that I wanted the button to display in the case that the widget was not locked. Apologies if that wasn't clear.

Oh, sorry. I misunderstood. Unfortunately, we cannot see, wether an item is locked at the moment. :(

**Comment by Pheels** [comment]
No problem, I appreciate your guidance nonetheless.

**Pull request merged**